app/src/main/res/values/strings.xml

```diff
@@ -210,11 +212,35 @@
 
     <!-- Spinner Download Size array-->
     <string-array name="download_size_array">
+        <item>Square</item>
+        <item>Thumbnail</item>
+        <item>Small</item>
+        <item>Medium</item>
+        <item>Large</item>
+        <item>Extra Large</item>
+    </string-array>
+
+    <!-- Spinner Download Size array (should stay untranslated) -->
+    <string-array name="download_size_array_values">
         <item>square</item>
         <item>thumb</item>
         <item>small</item>
         <item>medium</item>
         <item>large</item>
         <item>xxlarge</item>
     </string-array>
+
+    <!-- Color palette array -->
+    <string-array name="colors_array">
+        <item>Light</item>
+        <item>Dark</item>
+        <item>Auto</item>
+    </string-array>
+
+    <!-- Color palette array  (should stay untranslated) -->
+    <string-array name="colors_array_values">
```

Comment:
also translation=false
and consider using int values here, corresponding to the AppCompatDelegate.MODE_NIGHT_* (with a comment that refer to those constant names)

Reply:
Translatable label added in https://github.com/Piwigo/Piwigo-Android/pull/201/commits/b632c4ff9a3f9313a7c4e96a60897da733d3439f